cache/save.go

```diff
@@ -89,14 +89,24 @@ func NewSaver(
 
 // Save ...
 func (s *saver) Save(input SaveCacheInput) error {
+	saveStartTime := time.Now()
+	phaseStartTime := saveStartTime
+	defer func() {
+		saveTime := time.Since(saveStartTime).Round(time.Second)
+		s.logger.Donef("Save done in %s", saveTime)
+	}()
+
 	config, err := s.createConfig(input)
 	if err != nil {
 		return fmt.Errorf("failed to parse inputs: %w", err)
 	}
+	phaseTime := time.Since(phaseStartTime).Round(time.Second)
+	s.logger.Donef("Config created in %s", phaseTime)
```

Comment:
Are these information relevant for everyone? Have you considered doing these in debug mode?

Reply:
Also, we have methods with T prefix like `TInfof()` that automatically print time, especially if you only want second-precision, they should be simpler